src/plans/plan.jl

```diff
@@ -35,6 +37,57 @@ function get_manopt_parameter(f, e::Symbol, args...)
 end
 get_manopt_parameter(f, args...) = nothing
 
+"""
+    get_manopt_parameter(element::Symbol; default=nothing)
+
+Access global [`Manopt`](@ref) parametersadressed by a symbol `element`.
+We first dispatch on the value of `element`.
+
+if the value is not set, `default` is returned
+
+The parameters are queried from the global settings using [`Preferences.jl`](https://github.com/JuliaPackaging/Preferences.jl),
+so they are persistent within your activated Environment.
+
+# Currenlty used settings
+
+`:Mode`
+the mode can be set to `"Tutorial"` to get several hints especially in scenarios, where
+the optimisation on manifolds is different from the usual “experience” in
+(classicall, Euclidean) optimization.
+Any other value has the same effect as not setting it.
+"""
+function get_manopt_parameter(
+    e::Symbol, args...; default=get_manopt_parameter(Val(e), Val(:default))
+)
+    return @load_preference("$(e)", default)
+end
+# Handle empty defaults
+get_manopt_parameter(::Symbol, ::Val{:default}) = nothing
+get_manopt_parameter(::Val{:Mode}, v::Val{:default}) = ""
+
+"""
+    set_manopt_parameter!(element::Symbol, value::Union{String,Bool})
+
+Set global [`Manopt`](@ref) parameters adressed by a symbol `element`.
+We first dispatch on the value of `element`.
+
+The parameters are stored to the global settings using [`Preferences.jl`](https://github.com/JuliaPackaging/Preferences.jl).
+
+Passing a `value` of `""` deletes an entry from the preferences.
+Whenever the `LocalPreferences.toml` is modified, this is also `@info`rmed about.
+"""
+function set_manopt_parameter!(e::Symbol, value::Union{String,Bool,<:Number})
+    if length(value) == 0
+        @delete_preferences!("$(e)")
```

Comment:
```suggestion
        @delete_preferences!(string(e))
```
This is, I think, a more standard way of converting symbols to strings.